ROMFS/px4fmu_common/init.d-posix/10030_px4vision

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# @name PX4Vision SITL
+# Holybro px4vision. Gazebo Only.
+#
+# @type Quadrotor
+#
+
+sh /etc/init.d/rc.mc_defaults
+
```

Comment:
This is being initialized as a standard quadcopter. Shouldn't this be capable of using VIO and avoidance?

Reply:
Done!